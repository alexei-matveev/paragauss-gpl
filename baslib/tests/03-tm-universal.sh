../bin/qm-find-basis fe def-SVP universal
