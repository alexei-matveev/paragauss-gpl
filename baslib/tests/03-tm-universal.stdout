&unique_atom_basisset
lmax_ob=2
lmax_ch=4
lmax_xc=0
lmax_pseudo=-1
zc=0
/
&unique_atom_basis
n_exponents=14
n_contracted_fcts=5
n_uncontracted_fcts=0
/
0.033706691021 0.091449828308 0.71772177325 1.7245228003 6.0653307408 10.44994371 14.83091301 65.732730112 127.25891928 191.0904399 587.55977067 2081.3505927 9147.8893982 60923.640643
0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.29308335984 0.0 0.38253069946 0.18884995009 0.054332554248 0.010958790038 0.0014302254466
0.0 0.0 0.0 0.0 0.45472347323 0.0 0.64387631332 0.0 -0.10964564925 0.0 0.0 0.0 0.0 0.0
0.0 0.0 0.44985492922 0.72164398156 0.0 -0.22539639952 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0
0.0 1.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0
1.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0
&unique_atom_basis
n_exponents=9
n_contracted_fcts=3
n_uncontracted_fcts=0
/
0.134915 0.56061284958 1.4924724132 3.871932799 7.634855789 20.614988935 57.547272758 182.15149714 773.43750995
0.0 0.0 0.0 0.0 0.34284148028 0.52700011047 0.26993651996 0.070029620575 0.0094325735144
0.0 0.23649365839 0.56842594005 0.33974402988 0.0 0.0 0.0 0.0 0.0
1.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0
&unique_atom_basis
n_exponents=5
n_contracted_fcts=2
n_uncontracted_fcts=0
/
0.3652439317 1.2129967888 3.6136457999 10.800067078 38.968133419
0.0 0.47745100883 0.36905479496 0.14858319982 0.027879664382
0.31418142304 0.0 0.0 0.0 0.0
&unique_atom_basis
n_exponents=0
n_contracted_fcts=0
n_uncontracted_fcts=0
/
&unique_atom_basis
n_exponents=19
n_contracted_fcts=8
n_uncontracted_fcts=0
/
0.0808724 0.1627967 0.3212932 0.6265715 1.2321766 2.531276 4.9921254 9.8648356 18.7270618 37.1905224 73.1574429 146.2270422 312.9849847 605.7548994 1195.0581566 2333.9778892 4961.4420234 10088.625256 22231.167535
0.0 0.0 0.0 0.0 0.0 0.0 0.0 -5.1127291 11.2685887 16.4954218 -1.5186939 -0.1052884 13.5974545 17.3953035 13.7667481 7.2162778 4.830656 0.561673 1.5463792
0.0 0.0 0.0 0.0 0.0 0.0 1.318303 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0
0.0 0.0 0.0 0.0 0.0 4.6550254 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0
0.0 0.0 0.0 0.0 1.3101855 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0
0.0 0.0 0.0 0.1513152 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0
0.0 0.0 0.0845107 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0
0.0 0.0694777 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0
0.0157801 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0
&unique_atom_basis
n_exponents=5
n_contracted_fcts=5
n_uncontracted_fcts=0
/
0.1046543 0.3820476 1.3971534 5.1364325 19.0489263
0.0 0.0 0.0 0.0 -0.0062879
0.0 0.0 0.0 -0.0049488 0.0
0.0 0.0 0.0270354 0.0 0.0
0.0 -0.0123603 0.0 0.0 0.0
0.0199595 0.0 0.0 0.0 0.0
&unique_atom_basis
n_exponents=5
n_contracted_fcts=5
n_uncontracted_fcts=0
/
0.1829694 0.5410331 1.5953692 4.6650265 13.4494385
0.0 0.0 0.0 0.0 -0.0256685
0.0 0.0 0.0 -0.0262829 0.0
0.0 0.0 -0.0486537 0.0 0.0
0.0 -0.0025964 0.0 0.0 0.0
0.0085806 0.0 0.0 0.0 0.0
&unique_atom_basis
n_exponents=3
n_contracted_fcts=2
n_uncontracted_fcts=0
/
0.0884909 0.6208564 4.430072
0.0 0.0038112 -0.0066965
-9.53e-5 0.0 0.0
&unique_atom_basis
n_exponents=3
n_contracted_fcts=3
n_uncontracted_fcts=0
/
1.0943052 3.6312678 11.9744409
0.0 0.0 -0.0555001
0.0 -0.056053 0.0
-0.0150572 0.0 0.0
&unique_atom_basis
n_exponents=0
n_contracted_fcts=0
n_uncontracted_fcts=0
/
&unique_atom_basis
n_exponents=0
n_contracted_fcts=0
n_uncontracted_fcts=0
/
